FirebaseAuth/Sources/Backend/RPC/Proto/TOTP/FIRAuthProtoStartMFATOTPEnrollmentResponseInfo.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FirebaseAuth/Sources/Backend/RPC/Proto/FIRAuthProto.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRAuthProtoStartMFATOTPEnrollmentResponseInfo : NSObject <FIRAuthProto>
+
+@property(nonatomic, copy, readonly, nonnull) NSString *sharedSecretKey;
```

Comment:
For parity with the remaining codebase, there are no docs in `Backend/RPC/Proto` files. 
For eg: [FIRAuthProtoFinalizeMFATOTPSignInRequestInfo.h](https://github.com/firebase/firebase-ios-sdk/pull/11128/files#diff-958bf308d5de6c0b749936427a214441678da6972b203d88a5bf4e7da46c7a9b), [FIRAuthProtoStartMFAPhoneRequestInfo.h
](https://github.com/firebase/firebase-ios-sdk/blob/totp-release-0413/FirebaseAuth/Sources/Backend/RPC/Proto/Phone/FIRAuthProtoStartMFAPhoneRequestInfo.h)
I think we can skip this for now and create an internal bug for adding documentation to all these files if necessary?

Reply:
For coding best practice and readability I think moving forward we should add doc as we code.  I am fine with creating a bug for now if it's too much.  Also, it might be easier to just copy the comments from backend proto.